### ISSUE TITLE
IIS Provider - BindingHostRequired

### ISSUE THREAD

**Issue by jbrozek** [ebekker/ACMESharp#299]
Looks like IisHelper.cs is missing the following in CreateSiteBinding 

```
//determine ssl flag
if (binding.BindingHostRequired.GetValueOrDefault() && GetIisVersion().Major >= 8)
	b.SetAttributeValue("sslFlags", 1);
else
	b.SetAttributeValue("sslFlags", 3);
```

This seems to be causing an issue when other SNI site bindings already exist.

**Comment by ebekker** [comment]
Good catch!  Would you like to submit a PR to fix this?

**Comment by jbrozek** [comment]
I attempted to but I'm not clear on how to get this project running locally to test it via powershell on my local machine.

**Comment by ebekker** [comment]
Were you able to clone and build the project locally?

**Comment by jbrozek** [comment]
Yes, I was able to pull it down and build most of the projects, testing projects all seem to have missing elements preventing build.  Am I missing a developer doc somewhere that I could read?  I didn't see one.

**Comment by ebekker** [comment]
No, there is no doc, what errors are you getting?  Let's start to document the missing pieces here.

**Comment by jbrozek** [comment]
![error](https://user-images.githubusercontent.com/7240311/32246361-b8cbce72-be4c-11e7-9443-a2ecb3e27282.png)

- Looks like the first 3 are coming from an nuget package that is having an issue, ManagedOpenSsl64.  Re-adding this (and going up a version 0.6.1.3) seems to fix this. Then the linked files under x64 need some work.
- The others are missing files

**Comment by ebekker** [comment]
OK, a lot of the unit tests are verifying the various providers work, and the ones that are broken above (missing) are related to the AWS provider.  So that requires that you have an AWS account that you point to.

For each of the missing files above, you'll find a corresponding `.sample` file.  If you have an AWS account, you can copy the sample files and update them with your own AWS info.  If you do not have an AWS account, then you won't be able to run those tests.

**Comment by jbrozek** [comment]
Great, given i clear the errors above, back to the unknown.  How do I test this code change (via powershell) on my local machine or a dev server?
